src/org/labkey/test/tests/SampleSetTest.java

```diff
@@ -122,12 +121,15 @@ private void doSetup()
         _containerHelper.createSubfolder(PROJECT_NAME, FOLDER_NAME, new String[]{"Experiment"});
         _containerHelper.createSubfolder(PROJECT_NAME, LINEAGE_FOLDER, new String[]{"Experiment"});
 
+        goToProjectHome();
         projectMenu().navigateToProject(PROJECT_NAME);
         portalHelper.addWebPart("Sample Sets");
 
+        goToProjectHome();
         projectMenu().navigateToFolder(PROJECT_NAME, FOLDER_NAME);
         portalHelper.addWebPart("Sample Sets");
 
+        goToProjectHome();
         projectMenu().navigateToFolder(PROJECT_NAME, LINEAGE_FOLDER);
```

Comment:
These `goToProjectHome` calls shouldn't be necessary. There's a bug in the project menu helper if they are.

Reply:
Correct. I have removed them. I had added them when testing something locally and forgot to remove. Done.